apps/site/scripts/lint-snippets/index.mjs

```diff
@@ -0,0 +1,96 @@
+import { readFile } from 'node:fs/promises';
+
+import { parse } from 'acorn';
+import { glob } from 'glob';
```

Comment:
why not use glob from node ?

Reply:
It's a experimental API and we already have the `glob` installed so

Reply:
Okay let's use it. And ofc when node api will became stable we will be able to use it